examples/integration-gin/main.go

```diff
@@ -14,5 +14,13 @@ func main() {
 		c.HTML(http.StatusOK, "", Home())
 	})
 
+	r.GET("/with-ctx", func(c *gin.Context) {
+		c.Render(http.StatusOK, &TemplRender{
```

Comment:
I would create a function for that.

Reply:
I would probably do it as well on my code but being an example, it seems that the abstraction could confuse people. Leaving it "open" and without that extra layer seems to me a better approach. It is easy enough to understand. That said, if the maintainers prefer that function in, I can set it up.

Reply:
I'm not a Gin expert, I've never actually used it. I have no idea what the best approach is here.

I agree with the concept of keeping things simple in example code though.

@SebastianStehle - do you have an example of what you're talking about? I don't understand what you mean.